runHarness/RunProcedures/RunProcedure.pm

```diff
@@ -226,7 +232,7 @@ sub followRunProgress {
 	my ($self, $tmpDir)         = @_;
 	my $console_logger = get_logger("Console");
 	my $rampUp         = $self->getParamValue('rampUp');
-	my $steadyState    = $self->getParamValue('steadyState');
+	my $steadyState = $self->getParamValue('qosPeriodSec');
```

Comment:
For fixed, this causes start/stop stats on hosts to span qos periods (if there are more than 1 -- otherwise the stop happens during rampdown).

Reply:
That is ok.  We just want to get stats for a time while in steadyState.  I am only collecting for one QoS period because the run may exit after the first period, and detecting that would be a more significant change.  
I would prefer that stats collection didn't extend into rampdown, but it is ok.  The rampDown doesn't actually ramp the load down, it just continues to run at the same load for a few minutes after the last QoS period to ensure that shut-down effects don't mess up the QoS stats.  I would like to change the name of the interval from rampDown, but it would need to be changes in a lot of places.